custom/icds_reports/utils/aggregation_helpers/aww_incentive.py

```diff
@@ -27,21 +27,23 @@ def aggregation_query(self):
           SELECT
             %(state_id)s AS state_id,
             %(month)s AS month,
-            awc_id,
-            block_id,
-            state_name,
-            district_name,
-            block_name,
-            supervisor_name,
-            awc_name,
-            aww_name,
-            contact_phone_number,
-            wer_weighed,
-            wer_eligible,
-            awc_num_open,
-            valid_visits,
-            expected_visits
-          FROM agg_ccs_record_monthly AS acm
+            awcm.awc_id,
+            awcm.block_id,
+            awcm.state_name,
+            awcm.district_name,
+            awcm.block_name,
+            awcm.supervisor_name,
+            awcm.awc_name,
+            awcm.aww_name,
+            awcm.contact_phone_number,
+            awcm.wer_weighed,
+            awcm.wer_eligible,
+            awcm.awc_num_open,
+            ccsm.valid_visits,
+            ccsm.expected_visits
+          FROM agg_ccs_record_monthly AS ccsm
+          INNER JOIN agg_awc_monthly as awcm
+          ON chm.month=acm.month AND chm.awc_id=awcm.awc_id
           WHERE acm.month = %(month)s AND acm.state_id = %(state_id)s and aggregation_level=5
```

Comment:
What does the query planner look like for this query vs if you had changed this to `ccsm.month = x AND ccsm.state_id = Y` ?

I've generally used WHERE clauses that reference teh table that's in `FROM` instead of (or in addition to) the tables that are JOINED later. I think I reasoned about it by thinking that postgres would first look at the FROM table first and filtering early is generally always better. As I'm typing this out, I'm not sure how much I've actually validated that hypothesis by using the query planner. Perhaps its smart enough to work in this case? (although I'm skeptical it would be able to assume `ccsm.state_id = state_id` from the expression `acm.state_id = state_id` because that's not in the join)